api/node/server/server.go

```diff
@@ -188,17 +187,15 @@ func (s *Server) PostPublishProtocol(
 	if err != nil {
 		return nil, err
 	}
-	proposal := &types.Proposal{}
-	codec.MustDecode(blob, proposal)
 
 	protocol := string(request.Protocol)
-	if protocol == "hare3" {
+	switch protocol {
```

Comment:
Why single case switch instead of a simple if?

Reply:
because there were 2 cases for a moment... :sweat_smile: